src/app/pages/users/user/settings/activity/activity-card/activity-card.component.ts

```diff
@@ -13,22 +12,24 @@ import { LanguagesService } from '../../../../../../@core/utils/languages.servic
 export class ActivityCardComponent {
   activities;
   activities$;
-  @ViewChild('rangepicker') rangepicker;
+  dateValue: FormControl = new FormControl();
+
 
   constructor(private userService: UserData, private route: ActivatedRoute) {
-    this.userService.getLoginActivity().subscribe(activities => {
-      this.activities = activities;
+    this.dateValue.valueChanges.subscribe(selectedDate=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
    this.dateValue.valueChanges.subscribe(selectedDate =>{
```